src/fparser/Fortran2003.py

```diff
@@ -2154,10 +2154,11 @@ class Binding_Attr(STRINGBase): # R453
     <binding-attr> = PASS [ ( <arg-name> ) ]
                      | NOPASS
                      | NON_OVERRIDABLE
+                     | DEFERRED
                      | <access-spec>
     """
     subclass_names = ['Access_Spec', 'Binding_PASS_Arg_Name']
-    def match(string): return STRINGBase.match(['PASS', 'NOPASS', 'NON_OVERRIDABLE'], string)
+    def match(string): return STRINGBase.match(['PASS', 'NOPASS', 'NON_OVERRIDABLE', 'DEFERRED'], string)
```

Comment:
Include 'DEFERRED' as valid binding attribute. Please tidy this line for pep8/pylint (the `return` should be on a new line).